GSA-TTS/FAC
# Search Updates (Bugfixes)

# Search Updates (Bugfixes)

Issue: Epic - https://github.com/GSA-TTS/FAC/issues/2600

## Changes:
1. Form reset buttons. On form reset, have JS clear out the form inputs before wiping them. 
2. Reset page number to one if it goes past the number of available pages. Fixes a situation where one navigates to page two, changes their search parameters for a smaller result set, and seeing an error for visiting a non-existent page two. 
3. Have JS pick the form out by its ID. Fixes a problem where JS finds no form if you're not signed in, which subsequently breaks pagination and the form reset.

## How to test:
1. Switch to this branch and run normally. Ensure there is data in your dissemination tables. See [here](https://github.com/GSA-TTS/FAC/pull/2683) for some details on how to do that. 
2. Make sure the form reset buttons work:
    * On page load (put stuff in, then hit reset)
    * After a search (search for something, then hit reset)
3. Make sure pagination works when you're not signed in.

## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
